enterprise/api/v1/views.py

```diff
@@ -214,10 +207,7 @@ def course_enrollments(self, request, pk):
         return Response(serializer.errors, status=HTTP_400_BAD_REQUEST)
 
     @method_decorator(require_at_least_one_query_parameter('permissions'))
-    @list_route(permission_classes=[
-        permissions.IsAuthenticated,
-        IsInEnterpriseGroup,
-    ])
+    @list_route(permission_classes=[permissions.IsAuthenticated, IsInEnterpriseGroup])
     def with_access_to(self, request, *args, **kwargs):  # pylint: disable=invalid-name,unused-argument
```

Comment:
i would say that we should delete this because we don't need it anymore, but i think the right approach would be to introduce a v2 enterprise api that doesn't include this endpoint.

Reply:
I am not sure what to do here. I think creating v2 is out of scope and also I am not sure what exactly we need to add into V2? Should we copy the existing V1 code to V2 and remove the `with_access_to` route?

Reply:
sorry, i should have clarified. this was meant to be informational/a note for the future. i don't know when the right time to create a v2 api is, but i just want us to start thinking about it.

Reply:
Thanks a lot for the clarification. So should I remove the endpoint and permission classes or leave them?

Reply:
leave this for now.